src/lua_gears.cc

```diff
@@ -30,6 +30,8 @@ static void raw_init(lua_State *L, const Ticket &t,
   lua_setfield(L, -2, "engine");
   LuaType<const string &>::pushdata(L, t.name_space);
   lua_setfield(L, -2, "name_space");
+  LuaType<const Ticket &>::pushdata(L, t);
+  lua_setfield(L, -2, "ticket");
```

Comment:
There is no need to add the field,

Reply:
OK , Remove it later, I did wrong at the first place.